examples/source/interactivity-dynamic-content/Infinite_Cards_Email.html

```diff
@@ -0,0 +1,123 @@
+<!---
+
+preview: amp4email
+skipValidation: 'true'
+author: fstanis
+
+--->
+
+<!--
+## Introduction
+
+This sample demonstrates how to display a feed of data, allowing the user to go through
```

Comment:
So, this is an interesting problem. Emails allow XHR forms with `method="get"`, but AMP for websites doesn't, which leads to unintended / undefined behavior, specifically, it falls victim to XHR caching.

In an email, you get a different image every time you press "Next". On the web, to achieve this, you'd have to add `<input type="hidden" name="r" value="RANDOM" data-amp-replace="RANDOM">` (which is invalid in emails).

I could also modify the photo-stream endpoint to make it support POST and then change the form method to that. This may be the easiest solution. Any thoughts on that?

Reply:
I still don't understand the problem you're solving.

Reply:
The idea is for the user to be able to refresh parts of an amp-list without refreshing the whole list.

I updated it to use post so it no longer has the caching issue and should work correctly now. PTAL.